data/json/items/tools.json

```diff
@@ -5461,7 +5461,7 @@
     "id": "oxygen_tank",
     "type": "TOOL",
     "name": "oxygen tank",
-    "description": "This is tank of compressed medical oxygen with a regulator and mask.  Commonly used in emergency situations, it provides immediate relief for asthma attacks or smoke inhalation, and can provide a brief burst of energy.",
+    "description": "This is tank of compressed medical oxygen with a regulator and mask.  Commonly used in emergency situations, it provides immediate relief for asthma attacks or smoke inhalation, and can provide a brief burst of energy. Can also provide air underwater.",
```

Comment:
Both of the medical tank descriptions still need to be fixed.

Reply:
good catch, it slipped my mind